api/src/database/helpers/sequence/dialects/postgres.ts

```diff
@@ -0,0 +1,13 @@
+import type { Knex } from 'knex';
+import { AutoSequenceHelper } from '../types.js';
+
+export class AutoIncrementHelperPostgres extends AutoSequenceHelper {
+	/**
+	 * Resets the auto increment sequence for a table based on the max value of the PK column.
+	 * We're assuming that the default sequence name is being used,
+	 * which is the `${tableName}_${columnName}_seq`.
+	 */
+	override resetAutoIncrementSequence(_table: string, _column: string): Knex.Raw | null {
```

Comment:
The underscore in the `_table` and `_column` parameters should be removed as they are not unused.

Reply:
oh true.. a leftover from another approach.. 👍